app/src/main/java/org/fossasia/openevent/general/utils/Utils.kt

```diff
@@ -77,6 +79,14 @@ object Utils {
         inputManager.hideSoftInputFromWindow(view.windowToken, InputMethodManager.SHOW_FORCED)
     }
 
+    fun setToolbar(activity: FragmentActivity?, title: String, hasUpEnabled: Boolean, show: Boolean = true) {
```

Comment:
```suggestion
    fun setToolbar(activity: FragmentActivity?, title: String, hasUpEnabled: Boolean = true, show: Boolean = true) {
```

And remove true from calling site

Reply:
Okay

Reply:
Done